src/interfaces/business-i.ts

```diff
@@ -1,13 +1,97 @@
+import type { CorpTypeCd, AmalgamationTypes, FilingTypes } from '@bcrs-shared-components/enums'
+import type { ApiDateTimeUtc, IsoDatePacific } from '@bcrs-shared-components/interfaces'
+
 import { BusinessStateE } from '~/enums/business-state-e'
-import { BusinessTypeE } from '~/enums/business-type-e'
+import { FilingSubTypeE } from '~/enums/filing-sub-type-e'
+import type { StateFilingHeaderI } from '~/interfaces/state-filing-i'
 
-export interface BusinessI {
+export interface SlimBusinessI {
   adminFreeze: boolean
   alternateNames: { operatingName: string }[]
   goodStanding: boolean
   identifier: string
   legalName: string
-  legalType: BusinessTypeE
+  legalType: CorpTypeCd
   state: BusinessStateE
   taxId?: string
+  inDissolution: boolean
+}
+
+export interface StateFilingI {
+  business: any
+  consentContinuationOut?: any
+  header: StateFilingHeaderI
+  dissolution?: any
+  restoration?: any
+  putBackOn?: any
+}
+
+export interface FilingTypeI {
+  displayName: string
+  feeCode: string
+  name: FilingTypes
+  type?: FilingSubTypeE
+}
+
+export interface AllowedActionsI {
+  digitalBusinessCard: boolean
+  filing: {
+    filingSubmissionLink: string
+    filingTypes: FilingTypeI[]
+  }
+}
+
+export interface AmalgamatedIntoI {
+  amalgamationDate: ApiDateTimeUtc
+  amalgamationType: AmalgamationTypes
+  identifier: string // eg, BC7654321
+  legalName: string
+}
+
+export interface BusinessWarningI {
+  code: string // FUTURE: use an enum
+  filing?: string // not used
+  message: string
+  warningType: WarningTypesE
+  data?: any // optional extra properties (eg, amalgamationDate)
+}
```

Comment:
The `WarningTypesE` here and the `WarningType` in the `warning-type-e.ts` are not consistent. Maybe we can use `WarningTypeE`?

Reply:
I renamed and reorganized this, nice spotting it. 
This happened as I introduced some of existing interfaces/enums from entities.